src/main/java/mate/academy/springbootwebgreqit/repository/BookPaginationAndSortingRepository.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.springbootwebgreqit.repository;
+
+import mate.academy.springbootwebgreqit.model.Book;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.repository.PagingAndSortingRepository;
+
+public interface BookPaginationAndSortingRepository extends PagingAndSortingRepository<Book, Long> {
```

Comment:
Where did you use this interface and methods?

Reply:
Oh, I thought that I needed this class, but now I know that `BookRepository` already has these methods